docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,328 @@
+# High Availability
+
+## Set up etcd
+
+1. Create 3 Virtual Machines that follow [CoreOS' hardware recommendations](https://coreos.com/etcd/docs/latest/op-guide/hardware.html).
+   For the sake of simplicity, we will refer to them as `etcd0`, `etcd1` and `etcd2`.
+
+### create CA
+
+1. SSH into `etcd0` and run the following:
+
+```
+mkdir ~/certs
+cd ~/certs
+```
+```
+cat >ca-config.json <<EOL
```

Comment:
any less verbose way we could create these certs?

Reply:
we could use openssl directly, which would be succincter. I was under the impression cfssl is preferred though. WDYT?

Reply:
Are there not valid use cases where certs are not needed? I am thinking of on-premises installations where etcd would be running in a trusted environment? 

If you agree here, it would be worthwhile to mark SSL setup as "additional" rather than part of the "necessary minimum" and make the setup guide more modular so that this step can be skipped.

Reply:
I think SSL should always be strongly recommended. If folks want to have an insecure cluster, that's their prerogative, but I don't think we should structure our docs to facilitate that.

Reply:
I prefer verbosity on instructions here